File: src/main/scala/lesson/scala/file/FileWrite.scala
package lesson.scala.file

import java.io.PrintWriter

object FileWrite {
  def main(args: Array[String]) = {
    val lines = List("scala", "a", "b", "c", "d", "e")
    val writer = new PrintWriter("data/scala_out.txt")
    for(line <- lines){
      writer.println(line)
    }
    writer.close()
  }
}

File: src/main/java/lesson/java/file/FileWrite.java
package lesson.java.file;

import java.io.PrintWriter;

public class FileWrite {
    public static void main(String args[]){
        try{
            String lines[] = {"java", "a", "b", "c", "d", "e"};
            PrintWriter writer = new PrintWriter("data/java_out.txt");
            for(String line: lines){
               writer.println(line);
            }
            writer.close();
        }catch(Exception e){
            System.out.println(e.toString());
        }
    }
}
